Styles/navigation.css

```diff
@@ -47,7 +48,7 @@
     font-size: 1.5rem !important;
```

Comment:
Is any specific reason for important

Reply:
yes, google icons not allowing any change in size, tried few things didn't work. Only using it for google icons size over ride no other place.